docs/_docs/extensions-and-integrations/spring/spring-tx.adoc

```diff
@@ -0,0 +1,471 @@
+            name="configurationPath"
+            value="<path to an Apache Ignite configuration XML file (path can be absolute or relative to `IGNITE_HOME`)"/>
+    </bean>
+
+    <-- Use annotation-driven transaction configuration. -->
+    <tx:annotation-driven/>
+</beans>
+----
+--
+
+[NOTE]
+====
+If no connection configuration is specified, `SpringTransactionManager` will use Apache Ignite node instance with the
+default name. If it does not exist, an exception will be thrown during the `SpringTransactionManager` startup.
+====
+
+[IMPORTANT]
+====
+[discrete]
+No matter what configuration approach you chose, the same Apache Ignite node instance must be used to both initialize the
+transaction manager and perform transactional operations. In case Apache Ignite node was started by the transaction manager,
+you can obtain its instance in your application through `Ignition.ignite("<name of the Apache Ignite node instance>");`.
+====
+
+
+=== Transaction concurrency mode configuration
+
+The transaction concurrency mode configuration defines the
+https://ignite.apache.org/docs/latest/key-value-api/transactions#concurrency-modes-and-isolation-levels[Apache Ignite
+transaction concurrency mode, window="_blank"]
+that a manager will apply to all transactions that it processes.
+
+[tabs]
+--
+tab:Java[]
+[source,java]
+----
+@Configuration
+@EnableTransactionManagement
+public class SpringApplicationConfiguration {
+    @Bean
+    public SpringTransactionManager transactionManager() {
+        SpringTransactionManager mgr = new SpringTransactionManager();
+
+        mgr.setTransactionConcurrency(TransactionConcurrency.OPTIMISTIC);
+
+        return mgr;
+    }
+}
+----
+tab:XML[]
+[source,xml]
+----
+<beans xmlns="http://www.springframework.org/schema/beans"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xmlns:tx="http://www.springframework.org/schema/tx"
+       xsi:schemaLocation="
+           http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd
+           http://www.springframework.org/schema/tx http://www.springframework.org/schema/tx/spring-tx.xsd">
+    <-- Provide Apache Ignite instance name. -->
+    <bean id="transactionManager" class="org.apache.ignite.transactions.spring.SpringTransactionManager">
+        <!-- Here is appropriate connection configuration. -->
+        <property name="transactionConcurrency" value="OPTIMISTIC"/>
+    </bean>
+
+    <-- Use annotation-driven transaction configuration. -->
+        <tx:annotation-driven/>
+</beans>
+----
+--
+
+[NOTE]
+====
+If no transaction concurrency mode configuration is specified, `PESSIMISTIC` concurrency mode will be used.
+====
+
+In case you need to support multiple transaction concurrency modes in your application, you need to create separate
+transaction managers for each transaction concurrency mode that you need, and for each Spring transaction specify the
+manager that will process it.
```

Comment:
```suggestion
In case you need to support multiple transaction concurrency modes in your application, create separate transaction managers for each transaction concurrency mode that you need, and for each Spring transaction specify the manager that will process it.
```